loading.bs

```diff
@@ -1558,6 +1561,21 @@ A [=request=] |browserRequest| <dfn lt="matches the stored exchange|doesn't matc
 1. If |browserRequest|'s [=request/url=] is not [=url/equal=] to
     |storedExchange|'s [=exchange/request URL=], return "mismatch".
 
+1. If |browserRequest|'s [=request/header list=] contains a [=header=]
+    [=header/named=] \``Cookie`\` and |storedExchange|'s [=exchange/response=]'s
+    [=response/header list=] contains a [=header=] [=header/named=] \``Vary`\`,
+    then:
+
+    1. Let |fieldValues| be the [=list=] containing the elements corresponding to
+        the [=http/field-values=] of the [=Vary=] header.
+
+    1. [=list/For each=] |fieldValue| of |fieldValues|:
+
+        1. If |fieldValue| matches "`Cookie`", return "mismatch".
```

Comment:
Should you link to https://infra.spec.whatwg.org/#byte-case-insensitive for a definition of `matches`?